modules/client-configuration/pages/vhm-caasp.adoc

```diff
@@ -26,18 +26,22 @@ When you have added the {caasp} nodes to {productname}, you will be able to:
 * Perform configuration management operations against the nodes, including using Salt states.
 * Assign different set of channels to clusters using channel staging and lifecycle management filters.
 
-Updates related to {k8s} are managed using the ``skuba`` tool.
+Updates related to {k8s} are managed using the ``skuba-update`` tool.
 For more information, see https://documentation.suse.com/suse-caasp/4/html/caasp-admin/_cluster_updates.html.
 
-////
-Waiting on clarification of this. LKB 2019-11-11
 [WARNING]
 ====
-Do not use {productname} to perform these actions on any {caasp} nodes:
-
-* Install, upgrade, or removal of ``kubernetes-kubeadm``, ``kubernetes-kubelet``, ``kubernetes-client``, ``cri-o``, or ``cni-plugins``
-* Scheduling and executing a node reboot
-
-These actions are not supported.
+Do not use Salt, {productname} (either via UI or API) to perform these actions on any {caasp} nodes:
+
+* Apply a patch (if the patch is marked as interactive)
+* Mark a system to automatically install patches
+* Perform an SP migration
+* Reboot a node
+* Issue any power management action via Cobbler
+* Installing a package if it breaks or conflicts the `patterns-caasp-Node-x.y`
+* Removing a package if it breaks or conflicts or is one of the packages related with the `patterns-caasp-Node-x.y`
+* Upgrading a package if it breaks or conflicts or is one of the packages related with the `patterns-caasp-Node-x.y`
+
+{productname} will not stop you from issuing these operations.
+Issuing those operation may render your {caasp} cluster unusable.
```

Comment:
I wonder if it would make more sense to put this in an admonition. I feel like listing a things of what not to do could be risky, as people tend to skim over stem sentences and jump straight to bullet points. I'm not sure we can trust that people will not read this as "oh, yeah, I can reboot nodes!"

Reply:
Yep, that works :)